packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -32,25 +32,6 @@ import { SidebarNavigationContext } from '../sidebar';
 
 const { useHistory, useLocation } = unlock( routerPrivateApis );
 
-function getBackPath( params ) {
```

Comment:
This is nice, thank you.

What do you think of collocating the backPath info in the router and always pass it down as a prop to the sidebar?

```
if ( path === '/page' ) {
   return {
     sidebar: <SidebarNavigationScreen backPath={ { path: '/page', postId } } />,
     // ....
   };
}
```

It's not big of a difference, really, given that it'd now behave like a prop to the sidebar component. This slight change just means that we still have a centralized place (the router) with all the routing information, so it's easier maintenance. When we open this for extension or introduce an API for route registration, it may be good to provide all routing logic in that API and not require clients to provide the backpath separately, as part of their Sidebar UI component.

Reply:
That's a great idea, I implemented it. Now all the wiring between paths and rendered components is concentrated in the router, including the backpaths.